### DISCUSSION TITLE
使用 Arc 浏览器自定义网页 | Super Blog

### DISCUSSION THREAD

**Discussion by giscus** [SuperPung/Blog#15] [General]
# 使用 Arc 浏览器自定义网页 | Super Blog

方便、快捷地自定义网页，无需安装任何插件。

https://superpung.com/arc-boost/

**Comment by zdiovo** [comment]
我也想体验一下arc浏览器，但是需要邀请码，求一个邀请码，如果看到了请发我：ikingfly@proton.me

**Reply by superpung** [reply]
done

**Comment by superpung** [comment]
hey, here’s an invite to Arc, the browser I was telling you about!

https://arc.net/gift/11c57ea0

**Reply by zdiovo** [reply]
收到，谢谢啦
